test/WebJobs.Script.Tests.Integration/TestScripts/Node/functions.tests.js

```diff
@@ -151,7 +151,9 @@ describe('functions', () => {
         context = {
             _inputs: [],
             bindings: {},
-            log: (message) => logs.push(message),
+            log: {
```

Comment:
with my suggestion above, this change shouldn't be needed, right? since log remains a function